### DISCUSSION TITLE
Anybody use Winter with Octane?

### DISCUSSION THREAD

**Discussion by fpolli** [wintercms/winter#1198] [Q&A] [answered]
I am working on a Laravel app that is based on FrankenPHP/Octane and wonder if anyone is using Winter with Octane. What issues would I need to look out for?

**Comment by LukeTowers** [comment, marked as answer]
@fpolli I haven't tried running it on Octane although that is definitely something I would love to support. Main issues are just going to be potential memory leaks and scope breaks; you'll need to make sure that the various singletons / manager classes in the core are reset between requests along with the various extra caches that we have (like the DB memory cache, the caches in the Halcyon Datasources, etc).

Keep us posted on any issues you have in a draft PR and I'll do my best to help resolve them so that we can get Winter supporting Octane!

**Reply by fpolli** [reply]
Thank you for the response.
